everyvoice/text/features.py

```diff
@@ -1,69 +1,32 @@
-from typing import List
-from unicodedata import normalize
-
 import numpy as np
 import numpy.typing as npt
 from panphon import FeatureTable
 
 from everyvoice.config.text_config import TextConfig
 
-N_PHONOLOGICAL_FEATURES = 39
+N_PHONOLOGICAL_FEATURES = 43
 
 
-# TODO: support primary and secondary stress
 class PhonologicalFeatureCalculator:
     def __init__(self, text_config: TextConfig, punctuation_hash: dict):
         self.config = text_config
         self.punctuation_hash = punctuation_hash
         self.feature_table = FeatureTable()
 
-    def get_tone_features(self, text: List[str]) -> npt.NDArray[np.float32]:
-        # TODO: sort out how to define encoding of tone features
-        """Return Wang (1967) style tone features.
-            - Contour
-            - High
-            - Central
-            - Mid
-            - Rising
-            - Falling
-            - Convex
+    def mask_token(self):
+        return self.get_features(["[MASK]"])[0]
 
-        *If your language uses phonemic tone you MUST amend this function to match your language
-        Panphon does not use these features.*
+    def pad_token(self):
+        return self.get_features(["[PAD]"])[0]
 
-        Args:
-            text (list(str)): segmented phones
-        """
-        tone_features = []
-        high_tone_chars = [
-            normalize("NFC", x)
-            for x in [
-                "áː",
-                "á",
-                "ʌ̃́ː",
-                "ʌ̃́",
-                "éː",
-                "é",
-                "íː",
-                "í",
-                "ṹː",
-                "ṹ",
-                "óː",
-                "ó",
-            ]
-        ]
-        low_tone_chars = [
-            normalize("NFC", x) for x in ["òː", "ũ̀ː", "ìː", "èː", "ʌ̃̀ː", "àː"]
-        ]
-        for char in text:
-            char = normalize("NFC", char)
-            if char in high_tone_chars:
-                tone_features.append([-1, 1, -1, -1, -1, -1, -1])
-            elif char in low_tone_chars:
-                tone_features.append([-1, -1, -1, -1, -1, -1, -1])
-            else:
-                tone_features.append([0, 0, 0, 0, 0, 0, 0])
-        return np.array(tone_features, dtype=np.float32)
+    def cls_token(self):
+        return self.get_features(["[CLS]"])[0]
+
+    def sep_token(self):
+        return self.get_features(["[SEP]"])[0]
+
+    def unk_token(self):
+        return self.get_features(["[UNK]"])[0]
```

Comment:
These five functions aren't used anywhere in the code base. Why implement them?

Reply:
They're used in PF-BERT and possibly with other BERT-style text encoding